g1ant-language/commands/checkversion.md

```diff
@@ -0,0 +1,31 @@
+# checkversion
+
+**Syntax:**
+
+```G1ANT
+checkversion path ‴‴ result ‴‴ assembly ‴‴ if ‴‴ errorjump ‴‴ errormessage ‴‴
```

Comment:
to be consistent, let's remove
"result ‴‴ assembly ‴‴ if ‴‴ errorjump ‴‴ errormessage ‴‴" part
because these are optional arguments :)

Reply:
let's leave it for now. This file needs more updates. Will be done in the future